### ISSUE TITLE
Design for homologous polymers？

### ISSUE THREAD

**Issue by Mistletoe-git** [dauparas/LigandMPNN#40]
Hello, thank you for your excellent work.

I am using LigandMPNN to design some sites on a polymer protein, and because of the symmetry, I want to redesign the specified sites on all chains the same, so I try to use the following parameters:

```
--redesigned_residues "E135 E178 E179 E244"
--homo_oligomer 1
```

But doing so only changes the sites on the E chain.

So maybe I should use --symmetry_residues and --symmetry_weights? But I don't really know how to do that, because I don't quite understand what the example --symmetry_residues "C1,C2,C3|C4,C5|C6,C7" means (forgive me for being a newbie).

So can you or someone else in the community give me some help? Thank you very much!

**Comment by amelie-iska** [comment]
So, for ProteinMPNN, LigandMPNN, etc. the predicted logits for tied positions are averaged to
get a single probability distribution from which amino acids are sampled. You can weight these if you like. The explanation is actually in [the original ProteinMPNN paper](https://www.biorxiv.org/content/10.1101/2022.06.03.494563v1). Have a look at Figure 3(G) for a visual illustration. 

I think the thing to keep in mind is that ProteinMPNN, LigandMPNN, etc. are all *autoregressive* with a *random decoding order*, and tying residues together (from a single chain or multiple chains) means the tied residues get decoded at the same time (and are sampled from the same distribution). Good luck!

**Comment by agheeraert** [comment]
Hi, I'm currently working with a dimer (two chains A and B). Indeed if I use your options only chain A is redesigned:
```
--redesigned_residues "A135 A178 A179 A244"
--homo_oligomer 1
```
However, using explicitely all chain names solves the issue:
```
--redesigned_residues "A135 A178 A179 A244 B135 B178 B179 B244"
--homo_oligomer 1
```
I'm not entirely sure what is the behavior of the first options but the second syntax appears be the correct one. I'd suggest you creating a shell variable for redesigned residues to avoid any input errors.
`redesigned_residues=$(echo {A,B}{135,178,179,244})`
